src/pykeen/models/unimodal/rotate.py

```diff
@@ -85,14 +86,19 @@ def __init__(
     ) -> None:
         super().__init__(
             triples_factory=triples_factory,
-            embedding_dim=2 * embedding_dim,
             loss=loss,
             preferred_device=preferred_device,
             random_seed=random_seed,
             regularizer=regularizer,
-            entity_initializer=xavier_uniform_,
-            relation_initializer=init_phases,
-            relation_constrainer=complex_normalize,
+            entity_representations=EmbeddingSpecification(
+                embedding_dim=2 * embedding_dim,
```

Comment:
Add comment that this is for complex numbers.

Didn't we also have support for passing `embedding_dim` and dtype `complex` which would get translated internally? This might be even better for exchanging with "real" complex tensors once the support is there.

Reply:
yes but that will require changing implementation, so it will also be next PR

Reply:
we actually already did this in #107